test/verify/check-system-info

```diff
@@ -380,5 +381,48 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+class TestPcp(packagelib.PackageCase):
+
+    @skipImage("cockpit-system doesn't have the install-on-demand feature", "rhel-7-5")
+    @skipImage("doesn't have pcp", "debian-stable")
+    @skipImage("https://bugzilla.redhat.com/show_bug.cgi?id=1559414", "rhel-x")
+    def testEnablePcpLink(self):
+        m = self.machine
+        b = self.browser
+
+        # the atomics don't have pcp and can't install additional software
+        if m.atomic_image:
+            self.login_and_go("/system")
+            b.wait_present("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#system-information-enable-pcp")
+            return
+
+        m.execute("pkcon remove -y pcp")
+
+        # the offer to install it should be visible
+        self.login_and_go("/system")
+        b.wait_present("#server-pmlogger-onoff-row")
+        b.wait_not_visible("#server-pmlogger-onoff-row")
+        b.wait_visible("#system-information-enable-pcp")
+        b.wait_in_text("#system-information-enable-pcp", "Enable persistent metrics…")
+        b.click("#system-information-enable-pcp-link")
+        b.wait_present("#cockpit_modal_dialog")
+        b.wait_in_text("#cockpit_modal_dialog .modal-header", "Install Software")
+
+        pmlogger = {
+            "/usr/bin/pmlogger": "",
+            "/etc/systemd/system/pmlogger.service": "[Service]\nExecStart=/usr/sbin/true"
```

Comment:
This should be /bin/true, this path does not exist.

Reply:
Ugh, how did that happen :roll_eyes: Sorry